frontend/src/routes/create-goals/+page.svelte

```diff
@@ -95,211 +80,379 @@
       </div>
     </div>
 
-    <div class="grid grid-cols-1 md:grid-cols-2 gap-5">
-      <!-- Specific card -->
-      <div class="card">
-        <figure>
-          <figcaption class="p-4 card-body">
-            <h2 class="card-title">Specific</h2>
+    <!-- Specific card -->
+    <div class="card w-full">
+      <figure>
+        <figcaption class="p-4 card-body flex flex-col">
+          <h2 class="card-title mb-2">Specific</h2>
+          <div class="flex flex-col md:flex-row w-full">
             <input
               id="specific"
-              class="shadow appearance-none border rounded w-full py-2 px-3 leading-tight focus:outline-none focus:shadow-outline"
+              class="shadow appearance-none border rounded-xl w-full py-2 px-3 leading-tight focus:outline-none focus:shadow-outline flex-grow mb-2 md:mb-0"
               type="text"
               placeholder="AchievAIm's Specific suggestion"
               bind:value={goal.specific}
             />
-            <div class="mt-3">
-              <button class="btn btn-primary">Keep Specific Suggestion</button>
-            </div>
-          </figcaption>
-        </figure>
-      </div>
+            <label
+              class="cursor-pointer label flex items-center md:ml-2 mt-2 md:mt-0 w-full md:w-auto justify-end"
+            >
+              <input type="checkbox" class="toggle toggle-primary" checked />
```

Comment:
These will all need to default to unchecked, otherwise when the generate button is clicked these will never get set because they will be locked.

```suggestion
              <input type="checkbox" class="toggle toggle-primary" />
```